pkgs/tools/networking/networkmanager_dmenu/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, glib, fetchFromGitHub, networkmanager, python3Packages
+, gobjectIntrospection, dmenu }:
+
+let inherit (python3Packages) python pygobject3;
+in stdenv.mkDerivation rec {
+  name = "networkmanager_dmenu-${version}";
+  version = "unstable-2017-04-13";
```

Comment:
I don't want to get into the package naming / versioning, but I'd just like to point out that Nix (as in "nix-env -u pkg") will see this as name=networkmanager_dmenu-unstable, version=2017-04-13. So you might want to move that "unstable" string into the name attribute, so that Nix and the attributes agree with each other.

(Nix doesn't look at the version attribute, it parses version info from the package name.)